src/Tests/PostfinancePaymentTest.php

```diff
@@ -26,7 +29,7 @@ class PostfinancePaymentTest extends WebTestBase {
     'payment_postfinance_test',
     'node',
     'field_ui',
-    'config'
+    'config',
```

Comment:
we don't need the config module in the test, that's just for configuration import/export stuff.